src/Microsoft.ML.Tokenizers/Tokenizer.cs

```diff
@@ -136,6 +137,76 @@ public int CountTokens(string text, bool considerSpecialTokens = true)
             return idsCount;
         }
 
+        /// <summary>
+        /// Find the maximum encoding capacity within the input text without surpassing the token limit.
+        /// </summary>
+        /// <param name="text">The text to encode.</param>
+        /// <param name="maxTokenCount">The maximum token count to limit the encoding capacity.</param>
+        /// <param name="fromStart">Indicate if want to trim from the start of the text.</param>
```

Comment:
As opposed to from the end?

Reply:
oops, initially I was not using Trim term and was just counting. After I renamed the method with trim now we need to rename this parameter.